.github/workflows/tests.yaml

```diff
@@ -41,6 +41,9 @@ jobs:
         # Don't build pyopencl as that is difficult
         cat myreq.txt | sed /pyopencl/d > req.txt
         pip install -r req.txt
+    - name: Run update
+      run: |
+        ./pull-packages.sh -x
```

Comment:
Slightly concerned about this as `pull-packages.sh -x` often fails when git refuses to update packages that have been force-pushed.

Reply:
This runs directly after `install.sh` installation, so there shouldn't really be updates to the git packages hopefully. There will be a bunch of conda updates though.